pkg/controller/runtimeextractor/controller.go

```
@@ -0,0 +1,169 @@
+		case <-ctx.Done():
+			klog.Info("[RuntimeExtractorController]: Context Done")
+		}
+	}
+}
+
+func (re *runtimeExtractorController) handleConfigChange(ctx context.Context) {
+	klog.Info("[RuntimeExtractorController]: handleConfigChange")
+	cfg := re.config.Config()
+
+	if cfg.DataReporting.DisableRuntimeExtractor {
+		re.deleteDeployment(ctx)
```

golangci-lint (2.11.4)

[error] Error return value of `re.deleteDeployment` is not checked

(errcheck)